input/examples/encounter-consultation.xml

```diff
@@ -4,7 +4,7 @@
         <valueString value="Encounter - Consultation"/>
     </extension>
     <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-description">
-        <valueMarkdown value="This is a consultation encounter example using the *AU Core Encounter* profile.&#xa;&#xa;Patient: Fred Roberts"/>
+        <valueMarkdown value="This is a consultation encounter example for the *AU Core Encounter* profile."/>
```

Comment:
For all examples, to be consistent with AU Core:
1. Start with "Shows an example of <insert thing here> for the *<insert profile* profile. Patient <insert patient>."

Reply:
Reformatted

Reply:
Some were, some aren't